cli/cmd/uninstall/uninstall.go

```diff
@@ -392,13 +393,11 @@ func (c *Command) deleteSecrets(foundReleaseName, foundReleaseNamespace string)
 		return nil
 	}
 	for _, secret := range secrets.Items {
-		if strings.HasPrefix(secret.Name, foundReleaseName) {
-			err := c.kubernetes.CoreV1().Secrets(foundReleaseNamespace).Delete(c.Ctx, secret.Name, metav1.DeleteOptions{})
-			if err != nil {
-				return fmt.Errorf("deleteSecrets: error deleting Secret %q: %s", secret.Name, err)
-			}
-			secretNames = append(secretNames, secret.Name)
+		err := c.kubernetes.CoreV1().Secrets(foundReleaseNamespace).Delete(c.Ctx, secret.Name, metav1.DeleteOptions{})
+		if err != nil {
+			return fmt.Errorf("deleteSecrets: error deleting Secret %q: %s", secret.Name, err)
 		}
+		secretNames = append(secretNames, secret.Name)
```

Comment:
I would move the instantiation of `secretNames` to right above line 394, before you start the iteration. I would also change the name to `deletedSecrets` so it's more clear what you are listing.

Reply:
I lean towards secretNames rather than changing it to deletedSecrets because they are the names of secrets to delete, but haven't been deleted yet.